### PR TITLE
Artp 325/add price history

### PR DESCRIPTION
**Pull request by snony**

### CODE REVIEW

File: src/client/src/ui/spotTile/spotTileReducer.ts [hunk 1]
```diff
@@ -34,7 +34,7 @@ const spotTileReducer = (
     case TILE_ACTION_TYPES.SPOT_TILE_SUBSCRIBE:
       return state
     case TILE_ACTION_TYPES.SPOT_PRICES_UPDATE:
-      return { ...state, price: action.payload }
+      return { ...state, price: action.payload, historicPrices: [...state.historicPrices.slice(0), action.payload] }
```
Comment (spcfran) [inline src/client/src/ui/spotTile/spotTileReducer.ts]: what is the intention with `[...state.historicPrices.slice(0), action.payload]` ?
Note that `state.historicPrices.slice(0)` is an array with the same elements as `state.historicPrices`, which is the same effect as spreading the array. Unless I'm missing something:
- if you intend to remove the first element of the array, should use `.slice(1)`
- if you intend to copy all elements and append the payload, just `[...state.historicPrices, action.payload]` will do

File: src/client/src/ui/spotTile/components/analyticsTile/AnalyticsTileChart.tsx [hunk 2]
```diff
@@ -1,40 +1,44 @@
...
-            <ReferenceLine y={0} {...lineProps} />
-          </LineChart>
-        </ResponsiveContainer>
-      </AnalyticsTileChartWrapper>
-    )
-  }
+interface Props {
+  history: SpotPriceTick[]
+}
+const AnalyticsTileChart: FC<Props> = ({ history }) => {
```
Comment (spcfran) [inline src/client/src/ui/spotTile/components/analyticsTile/AnalyticsTileChart.tsx]: In some places throughout the project we are doing:
```typescript
import React from 'react'
const SomeComponent: React.FC<Props> ...
```
whereas in others
```typescript
import React, { FC } from 'react'
const SomeComponent: FC<Props> ...
```
The same applies to class components (`extends React.Component` VS `extends Component`)

It's not a huge deal but I think we should consistently follow one - I'd vote for the first one as it's currently the approach used in most places

File: src/client/src/ui/spotTile/spotTileReducer.ts (hunk 1 above)
Comment (snony) [reply src/client/src/ui/spotTile/spotTileReducer.ts]: Oups, thanks for spotting that out, I meant to use slice(1). As new price data is coming in, I am removing the first one. So, I am essentially moving historicPrices by one value to the left.

File: src/client/src/ui/spotTile/components/analyticsTile/AnalyticsTileChart.tsx (hunk 2 above)
Comment (bhavesh-desai-scratch) [reply src/client/src/ui/spotTile/components/analyticsTile/AnalyticsTileChart.tsx]: I like the latter

**Pull request merged**